### ISSUE TITLE
need ability to grab public keys / ECC support

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-gatewayrs#3]
In the full miners we have [this script](https://github.com/NebraLtd/hm-miner/blob/master/start-miner.sh) which pulls in the `public_keys` of the miner to a file `var/data/public_keys`

```
PUBLIC_KEYS=$(/opt/miner/bin/miner print_keys)
[ $? -ne 0 ] && exit 1
echo $PUBLIC_KEYS > /var/data/public_keys
```

We need this implementing in hm-gatewayrs so that hm-config and hm-diag can have access to the keys for proper onboarding.

This will need implementing in https://github.com/helium/gateway-rs first - which I do not believe it has been yet.

**Comment by shawaj** [comment]
Fixed in #5

**Comment by shawaj** [comment]
Ref https://github.com/helium/gateway-rs/pull/125
